### PR TITLE
Docker

### PR DESCRIPTION
**Pull request by MoisesTapia**
This contribution is with docker, which are install in many sistemas like windows or all linux

Feaures:

- [X] Run HiddenEye with docker
- [X] Make image of docker with HiddenEye
- [x] Expose of container to insternet

ps
start
URL

### CODE REVIEW

**Comment by sTiKyt** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 19
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DarkSecDevelopers/HiddenEye/pullRequest?prid=5911278&bid=19365036)

**Comment by sTiKyt** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 13
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DarkSecDevelopers/HiddenEye/pullRequest?prid=5911278&bid=19365036)

**Comment by sTiKyt** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 5
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DarkSecDevelopers/HiddenEye/pullRequest?prid=5911278&bid=19365036)

**Comment by sTiKyt** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DarkSecDevelopers/HiddenEye/pullRequest?prid=5911278&bid=19365036)

**Comment by sTiKyt** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DarkSecDevelopers/HiddenEye/pullRequest?prid=5911278&bid=19365036)

**Comment by sTiKyt** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DarkSecDevelopers/HiddenEye/pullRequest?prid=5911278&bid=19365036)

**Comment by sTiKyt** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DarkSecDevelopers/HiddenEye/pullRequest?prid=5911278&bid=19365036)

**Comment by sTiKyt** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DarkSecDevelopers/HiddenEye/pullRequest?prid=5911278&bid=19365036)

**Comment by MoisesTapia** [comment]
This is my contribution about of tool but in codacy exist some feature that my docker is correct like:

this part of code exist in my Dockerfile
```text
RUN apt-get update && apt-get install -y python \
 && apt-get clean \
 && rm -rf /var/lib/apt/lists/*
```
I declared the version of python that I will install
```text
RUN apt-get install python=2.7
```
this is  not  part of my code
```text
Defs/FeatureManager/EmailManager/emailconfig.py
```

I'll wait your feedback

File: Docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,24 @@
+FROM python:3.7 as builder
+
+WORKDIR /home/
+
+RUN git clone https://github.com/DarkSecDevelopers/HiddenEye
```
Comment (robbyoconnor) [inline Docker/Dockerfile:5]: Do `COPY .. .`

File: Docker/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,24 @@
+FROM python:3.7 as builder
```
Comment (robbyoconnor) [inline Docker/Dockerfile:1]: Why?

File: Docker/Dockerfile (hunk 1 above)
Comment (MoisesTapia) [reply Docker/Dockerfile]: I moved the docker-compose and Docker file to principal folder and in Docker it just are README.md for follow instructions

File: Docker/Dockerfile (hunk 2 above)
Comment (MoisesTapia) [reply Docker/Dockerfile]: I removed it the multistage

**Comment by sTiKyt** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DarkSecDevelopers/HiddenEye/pullRequest?prid=5911278&bid=19365036)

**Comment by sTiKyt** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DarkSecDevelopers/HiddenEye/pullRequest?prid=5911278&bid=19365036)

**Comment by MoisesTapia** [comment]
@robbyoconnor 
 Update the Dockerfile and fix the proble with apt

this in not part of my code :
Defs/FeatureManager/EmailManager/emailconfig.py
```python
gmail_account = "GMAILACCOUNT"
gmail_password = "GMAILPASSWORD"
recipient_email = "RECIPIENTEMAIL"
```
And this:
Recomedation:
```bash
RUN apt-get install -y --no-install-recommends python=2.7
```

My Code:
```bash
RUN apt-get update && \
    apt-get install -y --no-install-recommends python3.7 && \
    apt-get install -y python3-pip && \
```
I'll waiting for your feedback thanks

**Comment by sTiKyt** [comment]
You should be updating  DEV branch, not master one... Master branch is very different

**Comment by MoisesTapia** [comment]
Okay

**Pull request closed without merging**